bridgenode/genproofs.go

```diff
@@ -42,7 +42,7 @@ func BuildProofs(
 		fmt.Printf("not in %s, specify alternate path with -datadir\n.", dataDir)
 		return err
 	}
-	knownTipHeight = 729000
```

Comment:
@adiabat was this for testing purposes?

Reply:
Oops yeah, didn't want to wait for it to finish (took forever).  Can get rid of that.  Useful one-liner when you don't want to wait to test things though!